src/cerberus.py

```diff
@@ -15,8 +15,12 @@
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 
```

Comment:
We need to add a new line over here for the linters to pass ( expects two new lines before and after each function ). Run tox . from the base directory to make sure it passes. It's a manual process for now, @Yashashree1997 is working on adding a CI to do the checks for us.

Reply:
Do you plan to use Travis?

Reply:
Yes, that's the plan: https://github.com/openshift-scale/cerberus/issues/8.

Reply:
Weird. Running tox didn't give me any error.

Reply:
I might have missed something, my bad. Thanks for checking.